beacon-chain/core/epoch/epoch_operations_test.go

```diff
@@ -675,3 +669,51 @@ func TestInclusionDistance_NotFound(t *testing.T) {
 		t.Fatal("InclusionDistance should have failed")
 	}
 }
+
+func TestRandaoMix_Ok(t *testing.T) {
+	randaoMixes := make([][]byte, config.LatestRandaoMixesLength)
```

Comment:
where does config come from ?

Reply:
defined on top of `epoch_processing.go`:

`var config = params.BeaconConfig()`

makes everything looks a lot neater w/o the whole `params.BeaconConfig()`

Reply:
Ah , thanks. Didnt see that